### PR TITLE
Bridge Camera Fix

### PR DESCRIPTION
**Pull request by Hunttark**
I am an idiot.
When the first PR broke, I added/changed camera locations, and forgot that each variable had to be defined and cameras were not organized per area. This has been fixed, all cameras have been given unique/proper names. I knew I would have forgotten something. This also fixes the darkness in the COs office.

- I am dumb
- It should be fixed
- Shoot me?

### CODE REVIEW

**Comment by Hunttark** [comment]
!bugfix
!review

**Comment by Hunttark** [comment]
!bugfix

File: html/changelogs/Huntime - BridgeCameraFix.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - bugfix: "I am an idiot and forgot that cameras are personally defined in var not by area."
```
Comment (Ben10083) [inline html/changelogs/Huntime - BridgeCameraFix.yml:41]: I don't think you should have such personal messages in the changelog, just state what changed.

**Review by Ben10083 — CHANGES_REQUESTED** [review]

File: html/changelogs/Huntime - BridgeCameraFix.yml (hunk 1 above)
Comment (Hunttark) [reply html/changelogs/Huntime - BridgeCameraFix.yml]: > I don't think you should have such personal messages in the changelog, just state what changed.
Resolved. Thanks for the feedback.

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**